ml-agents/mlagents/trainers/trainer/rl_trainer.py

```diff
@@ -209,6 +209,7 @@ def _increment_step(self, n_steps: int, name_behavior_id: str) -> None:
         p = self.get_policy(name_behavior_id)
         if p:
             p.increment_step(n_steps)
+        self.stats_reporter.set_stat("Step", float(self.get_step))
```

Comment:
@ervteng Is there a place that I can collect more granular/frequent step updates?

Reply:
I think this is the right place if we care about synchronizing with the other metrics.

The other place would be in the AgentProcessor, but that is a bit decoupled from the other metrics as it would be added pre-trajectory assembly.

Reply:
At some point (e.g. if the inference and training are separate processes) it might be useful to emit both as different stats (e.g. `steps_executed` and `steps_processed`)